events/janus_wsevh.c

```diff
@@ -839,3 +840,55 @@ static int janus_wsevh_callback(struct lws *wsi, enum lws_callback_reasons reaso
 	}
 	return 0;
 }
+
+/**
+ * Implements the connecting attempt to the upstream websocket server
+ * sets the connection result (lws_client_connect_info) to static wsi
+ */
+static void janus_wsevh_connect_attempt(lws_sorted_usec_list_t* sul) {
+	struct lws_client_connect_info i = { 0 };
+	i.host = address;
+	i.origin = address;
+	i.address = address;
+	i.port = port;
+	i.path = path;
+	i.context = context;
+	if(!strcasecmp(protocol, "wss"))
+		i.ssl_connection = 1;
+	i.ietf_version_or_minus_one = -1;
+	i.client_exts = exts;
+	i.protocol = protocols[0].name;
+	JANUS_LOG(LOG_INFO, "WebSocketsEventHandler: Connecting to upstream websocket server %s:%d...\n", address, port);
+	wsi = lws_client_connect_via_info(&i);
+	if(!wsi) {
+		/* As we specified a callback pointer in the context the NULL result is unlikely to happen */
+		disconnected = janus_get_monotonic_time();
+		reconnect = TRUE;
+		JANUS_LOG(LOG_ERR, "WebSocketsEventHandler: Connecting to upstream websocket server %s:%d failed\n", address, port);
+		return;
+	}
+	reconnect = FALSE;
+}
+
+/**
+ * Adopts the reconnect_delay value in case of an error
+ * Increases the value up to JANUS_WSEVH_MAX_RETRY_SECS
+ */
+static void janus_wsevh_calculate_reconnect_delay_on_fail(void) {
+	if(reconnect_delay == 0)
+		reconnect_delay = 1;
+	else if (reconnect_delay < JANUS_WSEVH_MAX_RETRY_SECS) {
+		reconnect_delay += reconnect_delay;
+		if (reconnect_delay > JANUS_WSEVH_MAX_RETRY_SECS)
```

Comment:
The two `if` above have an extra space that per code style shouldn't be there.

Reply:
ack